statistics/selectivity.go

```diff
@@ -35,6 +35,8 @@ type exprSet struct {
 	mask int64
 	// ranges contains all the ranges we got.
 	ranges []*ranger.Range
+	// numCols is the number of columns contained in the index or column(which is always 1).
```

Comment:
If it is always 1, why don't use a const?

Reply:
It is always 1 for the column, while it could also greater than 1 for the index.

Reply:
gotcha.